src/components/Message/styles/Embed.css.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import baseStyles from '../../../styles/resets/baseStyles.css.js'
+import { BEM } from '../../../utilities/classNames'
+import { getColor } from '../../../styles/utilities/color.js'
+import { noteBoxShadow } from '../../../styles/mixins/noteStyles.css'
```

Comment:
Flow complains for this import path. Needs `.css.js`.
It still works.. just Flow yelling.

For some context, I previously didn't use `Flow` for the `.css.js`. I figured... these shouldn't have too much going on.

I was wrong.

I forgot how complex CSS could be (it's been a while since I wrote Seed-like functional Sass).

So there's inconsistencies between the `css.js` files.

I'm gonna do a cleanup to add Flow and fix import paths (in another PR)